app/src/main/java/org/mozilla/focus/browser/integration/BrowserToolbarIntegration.kt

```diff
@@ -63,6 +64,21 @@ class BrowserToolbarIntegration(
             }
 
             setOnUrlLongClickListener { onUrlLongClicked() }
+
+            icons = icons.copy(
+                trackingProtectionTrackersBlocked = AppCompatResources.getDrawable(
+                    context,
+                    R.drawable.mozac_ic_shield
+                )!!,
+                trackingProtectionNothingBlocked = AppCompatResources.getDrawable(
+                    context,
+                    R.drawable.mozac_ic_shield
+                )!!,
+                trackingProtectionException = AppCompatResources.getDrawable(
+                    context,
+                    R.drawable.mozac_ic_shield_disabled
+                )!!
+            )
```

Comment:
Instead of this, we could update the default in AC, just wanted to make sure we have these changes here regardless.

Reply:
That sounds good!